src/util.js

```diff
@@ -32,10 +32,15 @@ export const flattenDeep = (list /* : any[] */) /* : any[] */ =>
     list.reduce((memo, x) => memo.concat(Array.isArray(x) ? flattenDeep(x) : x), []);
 
 const UPPERCASE_RE = /([A-Z])/g;
-const MS_RE = /^ms-/;
-
-const kebabify = (string /* : string */) /* : string */ => string.replace(UPPERCASE_RE, '-$1').toLowerCase();
-export const kebabifyStyleName = (string /* : string */) /* : string */ => kebabify(string).replace(MS_RE, '-ms-');
+const UPPERCASE_RE_TO_KEBAB = (match /* : string */)  /* : string */ => `-${match.toLowerCase()}`;
+
+export const kebabifyStyleName = (string /* : string */) /* : string */ => {
+  const result = string.replace(UPPERCASE_RE, UPPERCASE_RE_TO_KEBAB);
+  if (result.slice(0, 3) === 'ms-') {
```

Comment:
I added a similar optimization to my importantify PR: https://github.com/Khan/aphrodite/pull/201/commits/bd0a575916b7210847147a79f775c24199b5572a